style/main.css

```
@@ -0,0 +1,299 @@
+span {
+  font-weight: 700;
+}
+
+p {
+  font-weight: 500;
+}
+
+a {
+  color: inherit;
+  text-decoration: none;
+}
+
+body {
+  background-color: #ffffff;
+  color: #374151;
+  font-size: 24px;
+}
+
+main {
+  margin-top: 70px;
+}
+
+nav {
+  position: fixed;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  top: 0;
+  width: 100%;
+  height: 70px;
+  background-color: #ffffff;
+  padding: 10px 200px 10px;
+}
+
+.panda-login {
+  cursor: pointer;
+  background-color: var(--brandblue);
+  border-radius: 8px;
+  padding: 12px 20px;
+  width: 128px;
+  height: 48px;
+  display: inline-flex;
+  justify-content: center;
+  align-items: center;
+  font-size: 16px;
+  font-weight: 600;
+  color: #ffffff;
+}
+
+.panda-banner {
+  height: 540px;
+  background-color: var(--maincolor);
+}
+
+.panda-banner span {
+  display: block;
+  color: #374151;
+}
+
+.panda-banner div {
+  display: flex;
+  align-items: flex-start;
+  flex-direction: column;
+  justify-content: center;
+  max-width: 1200px;
+  margin: auto;
+  height: 540px;
+  line-height: 56px;
+}
+
+#buypanda {
```

html/css 로 작업하는 일이 앞으로 흔하진 않겠지만, 
네이밍/케이싱(case, camelCase 등)/스타일적용을 위한 선택자(id, class)방식에서 일관성을 지켜주는 것이 좋을 거에요

```suggestion
.buy-panda {
```

물론 의미를 명확히 해주는 것도 미래에 유지보수비용을 아낄 수 있습니다.
여기서는 panda-banner에 이어서, '구매하러가기' 의 의미를 담아서, main-cta 라던가, main-landing-banner 등을 붙일 수있겠네요. *cta는 call to action입니다
```html
<section class="panda-banner main-landing">
```